aqi_monitor/aqi_monitor/settings.py

```diff
@@ -118,3 +119,11 @@
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 
 STATIC_URL = '/static/'
+
+
+# CELERY STUFF
```

Comment:
@rusyaevsa try to think, will undesrtanding of the configs below be changed if we remove this comment line?

Reply:
Fixed